ui/apps/platform/src/Containers/Violations/Details/DeploymentDetails.tsx

```diff
@@ -130,6 +150,46 @@ const DeploymentDetails = ({ deployment }) => {
                     <FlexItem>
                         <SecurityContext deployment={relatedDeployment} />
                     </FlexItem>
+                    <FlexItem>
+                        <Title headingLevel="h3" className="pf-u-my-md">
+                            Network Policy
+                        </Title>
+                        <Divider component="div" />
+                    </FlexItem>
+                    <FlexItem>
+                        <Card isFlat data-testid="network-policy">
+                            <CardBody>
+                                {namespacePoliciesList?.length > 0 ? (
+                                    <TableComposable aria-label="Simple table" variant="compact">
```

Comment:
You can omit `aria-label` prop when there is a heading element like h3 here for **All network policies**

For more information:
* https://www.patternfly.org/v4/accessibility/product-development-guide/#labeling-with-aria
* https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/attributes/aria-label

Reply:
Thanks, updated